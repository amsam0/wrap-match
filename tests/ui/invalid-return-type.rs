fn main() {}

#[wrap_match::wrap_match]
fn invalid_return_type() -> Option<()> {
    Some(())
}
