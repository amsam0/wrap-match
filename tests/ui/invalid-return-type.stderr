error: wrap_match currently only supports functions that return Results
 --> tests/ui/invalid-return-type.rs:4:29
  |
4 | fn invalid_return_type() -> Option<()> {
  |                             ^^^^^^
